database.py

```diff
@@ -0,0 +1,20 @@
+# from sqlalchemy import create_engine
+from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from settings import settings
+
+SQLALCHEMY_DATABASE_URL = settings.DATABASE_URL
+
+# engine = create_engine(
+engine = create_async_engine(
+    SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False}
```

Comment:
The `connect_args` parameter with `check_same_thread` is typically used for SQLite databases. Ensure that this configuration is appropriate for your database type, as it might not be needed for other databases.